mcu_janus.go

```diff
@@ -1231,7 +1251,11 @@ func (p *mcuJanusSubscriber) SendMessage(ctx context.Context, message *MessageCl
 			msgctx, cancel := context.WithTimeout(context.Background(), p.mcu.mcuTimeout)
 			defer cancel()
 
-			p.joinRoom(msgctx, callback)
+			if data.Update {
+				p.update(msgctx, callback)
```

Comment:
Do we need to handle the case that the subscriber did not join yet, i.e. if an `update` is received as first message?

Reply:
On one hand it could be argued that the first requested offer should be a "normal" one and that there is nothing to update until the connection is established; it should be the responsability of the client to ensure that the proper message is sent. On the other hand, _be conservative in what you send, be liberal in what you accept_, so it could be fine to handle it just for robustness. So I guess it depends on how much effort it would be needed to implement that in the signaling server.